src/main/java/com/box/sdk/BoxDeveloperEditionAPIConnection.java

```diff
@@ -93,22 +95,25 @@ public BoxDeveloperEditionConnection(String entityId, String entityType, String
         this.entityType = entityType;
         this.privateKey = privateKey;
         this.privateKeyPassword = privateKeyPassword;
+        this.encryptionAlgorithm = encryptionAlgorithm;
     }
 
     /**
-     * Creates a new Box Developer Edition connection for App Auth.
-     * @param enterpriseId       the enterprise ID to use for App Auth.
-     * @param clientId           the client ID to use when refreshing the access token.
-     * @param clientSecret       the client secret to use when refreshing the access token.
-     * @param privateKey         the private key corresponding to the public key configured with Box Developer Edition.
-     * @param privateKeyPassword the password for the private key.
-     * @return                   a new instance of BoxAPIConnection.
+     * Creates a new Box Developer Edition connection with enterprise token.
+     * @param enterpriseId          the enterprise ID to use for requesting access token.
+     * @param clientId              the client ID to use when exchanging the JWT assertion for an access token.
+     * @param clientSecret          the client secret to use when exchanging the JWT assertion for an access token.
+     * @param privateKey            the private key corresponding to public key configured with Box Developer Edition.
+     * @param privateKeyPassword    the password for the private key.
+     * @param  encryptionAlgorithm  specifies the encryption algorithm for generating JWT
```

Comment:
Extra space between `@param` and `encryptionAlgorithm`. Also needs a period at the end of the description.